# Request IG-calculation
geometry_msgs::TransformStamped position
string ig_id
--- 
float64 ig